src/components/NavBar.js

```diff
@@ -1,17 +1,21 @@
-import React from 'react';
+import React, { Component } from 'react';
 import styled from 'styled-components';
 import { Link } from 'react-router-dom';
 
 const NavBarContainer = styled.div`
   width: 100%;
   height: 4rem;
-  background: steelblue;
+  background: #26a69a;
   display: flex;
   justify-content: space-around;
```

Comment:
This will need to change in order for the settings items to be aligned to the top icons.